middlewares/userValidations.js

```diff
@@ -142,5 +142,23 @@ export default {
     const error = new Error('Invalid uuid user id param');
     error.status = 400;
     return next(error);
+  },
+
+  validateImageURL(req, res, next) {
+    let { imageURL } = req.body;
+    let message;
+    imageURL = imageURL && imageURL.trim();
+    if (
+      typeof imageURL === 'string' &&
+      !imageURL.includes('.jpeg') &&
```

Comment:
This is a nice touch. However, this means that if the url is something like `jpeg.www`, it will pass this test, and we know it won't be a proper url.

I am thinking about this though. Cos even if you use a proper matching regex or the string method `endsWith` (you can check out `Sting.prototype.endsWith` on MDN), it will still not completely be an accurate test, cos something like `ademola.jpeg` will pass and it may not be an actual existing image.

So I'd like you to check google for some javascript solutions to affirm that an image has a valid url or src link. If you don't see a very good solution, google `image url validator npm`. I am sure some decent options could crop up, and before installing any one, you can test it on repl.it to see that it actually works as expected.